pkg/cmd/auth/gitcredential/helper_test.go

```diff
@@ -8,12 +8,18 @@ import (
 	"github.com/cli/cli/v2/pkg/iostreams"
 )
 
+// why not just use the config stub argh
 type tinyConfig map[string]string
 
 func (c tinyConfig) GetWithSource(host, key string) (string, string, error) {
 	return c[fmt.Sprintf("%s:%s", host, key)], c["_source"], nil
 }
 
+func (c tinyConfig) Get(host, key string) (val string, err error) {
```

Comment:
Was adding this method necessary? The interface that this struct aims to satisfy only has one method: https://github.com/cli/cli/blob/227d0e001806b72683081521675b3dec83f51544/pkg/cmd/auth/gitcredential/helper.go#L16-L18